apcd-cms/src/taccsite_cms/urls_custom.py

```diff
@@ -1,3 +1,6 @@
 from django.urls import path, include
 
-custom_urls = []
+custom_urls = [
+    path('custom_example/', include('apps.custom_example.urls', namespace='custom_example')),
```

Comment:
**Suggestion.** Do not enable custom app, because:

- New dev can reference enabled apps and custom app instructions.
- https://github.com/TACC/Core-CMS-Custom/pull/9 updated documentation to show this missing step.

```suggestion
custom_urls = [
```